docs/public/GettingStarted.rst

```diff
@@ -48,5 +48,57 @@ related span is ended.
 
 The concept of an active span is important, as any span that is created
 without explicitly specifying a parent is parented to the currently
-active span.
+active span. A span without a parent is called root span.
 
+Create nested Spans
+^^^^^^^^^^^^^^^^^^^
+
+.. code:: cpp
+
+    auto outer_span = tracer->StartSpan("Outer operation");
+    auto outer_scope = tracer->WithActiveSpan(outer_span);
+    {
+        auto inner_span = tracer->StartSpan("Inner operation");
+        auto inner_scope = tracer->WithActiveSpan(inner_span);
+        // ... perform inner operation
+        inner_span->End();
```

Comment:
Is this call to `->End()` optional? The destruction of `inner_span` would call `End()`.

Reply:
It's optional, as ~Span destructor calls it anyway. But the `Span` object returned by `Tracer::StartSpan` is shared_ptr and in case instrumentation code is referring to this object at some other places, the ~Span destructor will be called when all the references are cleaned up. It's always good ( and no harm ) to call it explicitly to export the Span. And when the later destructor is called, it won't perform this again.